suisa_sendemeldung/suisa_sendemeldung.py

```diff
@@ -255,6 +255,36 @@ def merge_duplicates(data):
     return data
 
 
+def get_artist(music):
+    """Get artist from a given dict.
+
+    Arguments:
+        music: music dict from API
+
+    Returns:
+        artist: string representing the artist
+    """
+    artist = ""
+    if music.get("artists") is not None:
+        artists = music.get("artists")
+        if type(artists) == list:  # pragma: no cover
```

Comment:
```suggestion
        if isinstance((artists, list):  # pragma: no cover
```

no we really need the pragma?

Reply:
Not really, I honestly just copied from below (and went overboard with both :smile:)